src/java.base/share/classes/java/lang/Long.java

```diff
@@ -1164,6 +1206,9 @@ private LongCache() {}
         static final Long[] cache;
         static Long[] archivedCache;
 
+        @Stable
+        static final char[] HEX256;
+
```

Comment:
Builds are broken.

```suggestion
```

Reply:
Also the stable annotation is missing in the copy of this field in the cache class.